### ISSUE TITLE
RSEQC read_distribution division by zero

### ISSUE THREAD

**Issue by avilella** [ewels/MultiQC#1571]
### Description of bug

```
│ Traceback (most recent call last):                                                                                                                                                                                             │
│   File "/home/avilella/.local/lib/python3.6/site-packages/multiqc/multiqc.py", line 624, in run                                                                                                                                │
│     output = mod()                                                                                                                                                                                                             │
│   File "/home/avilella/.local/lib/python3.6/site-packages/multiqc/modules/rseqc/rseqc.py", line 65, in __init__                                                                                                                │
│     n[sm] = getattr(module, "parse_reports")(self)                                                                                                                                                                             │
│   File "/home/avilella/.local/lib/python3.6/site-packages/multiqc/modules/rseqc/read_distribution.py", line 62, in parse_reports                                                                                               │
│     pcts[pk] = (float(d[k]) / t) * 100.0                                                                                                                                                                                       │
│ ZeroDivisionError: float division by zero                                                                                                                                                                                      │
```

### File that triggers the error

_No response_

### MultiQC Error log

_No response_

**Comment by avilella** [comment]
Example input file:

```
Total Reads                   18719642
Total Tags                    0
Total Assigned Tags           0
=====================================================================
Group               Total_bases         Tag_count           Tags/Kb             
CDS_Exons           19599146            0                   0.00              
5'UTR_Exons         0                   0                   0.00              
3'UTR_Exons         13824187            0                   0.00              
Introns             0                   0                   0.00              
TSS_up_1kb          1599340             0                   0.00              
TSS_up_5kb          1599783             0                   0.00              
TSS_up_10kb         1599783             0                   0.00              
TES_down_1kb        28593532            0                   0.00              
TES_down_5kb        198190813           0                   0.00              
TES_down_10kb       410205813           0                   0.00              
=====================================================================
```

**Comment by ewels** [comment]
Looks like this is from `read_distribution`, not `inner_distance`

**Comment by ewels** [comment]
Fixed in 4e7dbb64e253b8fb43a8729d0c2eca9102f276a5 - thanks for reporting!
